docs/source/Regions2D_functionality.ipynb

```diff
@@ -0,0 +1,2342 @@
+{
```

Comment:
<p>I am confused between Hake and trawl: which one are you trying to show.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OSOceanAcoustics/echoregions/pull/184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>I am trying to show Trawl. I will correct this</p>